pom.xml

```diff
@@ -200,7 +200,7 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-failsafe-plugin</artifactId>
-                    <version>2.22.1</version>
+                    <version>3.0.0-M5</version>
```

Comment:
Why this change? This is not the scope of this issue. Please roll this change back, and apply it in a new PR with the corresponding issue.